src/Select/Select.hooks.ts

```diff
@@ -147,12 +152,21 @@ export const useSelect = ({
     []
   )
 
-  const handleOpen = React.useCallback(() => {
-    dispatch({ type: ActionType.Open })
+  const handleClick = React.useCallback(() => {
+      dispatch({ type: ActionType.Focus })
```

Comment:
Why Focus doesn't open in reducer ?

Reply:
It does, the line right below is another dispatch of `Open` action type

Reply:
Yes but why doing in with 2 dispatch instead of one ?